pyslackersweb/website/__init__.py

```diff
@@ -45,7 +36,7 @@ def app_factory() -> web.Application:
 
     # this ordering is important, before reordering be sure to check
     # for dependencies.
-    website.cleanup_ctx.extend([apscheduler, client_session, slack_client, background_jobs])
+    website.cleanup_ctx.extend([apscheduler, client_session, slack_client, redis, background_jobs])
```

Comment:
IMO we could already set `apscheduler` & `redis` context in the global app. They'll probably be used by sirbot or other apps in the future

Reply:
I agree, I punted on that as I need to follow up with some research of how to access things from a parent app in a sub-app signal/cleanup context.

I know during the request cycle we can do `request.config_dict` but haven’t had time to search for anything else. Explicitly passing the parent app in is one option but seems clumsy.

Reply:
So the documented in aio-libs/aiohttp#2413 is mostly detached from the `cleanup_ctx` concept that many use to manage the lifecycle of things like a database connection, scheduler, etc.

As a result, I am adding a bit of a hack that I am not sure I want yet...